client/components/document.vue

```diff
@@ -65,6 +78,17 @@
       });
     },
 
+    mounted() {
+      if (this.shouldEmbed) {
+        this.sendNewSizeToParentWindow();
+        window.addEventListener('resize', this.sendNewSizeToParentWindow);
```

Comment:
This is not good. This is not enough. The internal iframe window will never resize on its own. It is controlled by outside. What you have to observe is changes to content and respond to this. Probably with some mutation observer or something like this. This is why I suggested that we should probably just use a [library](https://github.com/davidjbradshaw/iframe-resizer).